core/primitives/src/sharding.rs

```diff
@@ -564,8 +560,9 @@ impl ShardChunkHeader {
     ) -> Result<(), BadHeaderForProtocolVersionError> {
         const BLOCK_HEADER_V3_VERSION: ProtocolVersion =
             ProtocolFeature::BlockHeaderV3.protocol_version();
+        #[allow(deprecated)]
         const CONGESTION_CONTROL_VERSION: ProtocolVersion =
-            ProtocolFeature::CongestionControl.protocol_version();
+            ProtocolFeature::_DeprecatedCongestionControl.protocol_version();
```

Comment:
Remove `CONGESTION_CONTROL_VERSION`. Assume protocol_version is always greater than `CONGESTION_CONTROL_VERSION`